1/test/run

```diff
@@ -102,12 +102,16 @@ function test_get_job() {
 }
 
 function run_tests() {
-  local name=$1 ; shift
+  local name=$1
   CONTAINER_ARGS="-u 10000:0"
+  if [ $# -gt 1 ];
+  then
+      CONTAINER_ARGS+=" -e OPENSHIFT_JENKINS_JVM_ARCH=i386 "
+  fi
```

Comment:
Didn't rework to exactly like the mongo run's (would have lead to a much broader structural change) but I reworked it to something I think is similar.  Changes pushed.